mine/models/stg_customers.sql

```diff
@@ -0,0 +1,15 @@
+with (
+    select * 
+    from {{source('ecommerce', 'customers')}}
+    ) as source 
+    ,( select 
+       
+    customer_id,
```

Comment:
stylistic point: [this is how we order columns](https://app.getguru.com/card/cM54ddki/Ordering-Columns-in-dbt)